cea/tests/test_district_cooling.py

```diff
@@ -0,0 +1,200 @@
+"""
+Test cea.optimization.slave.cooling_main.py, which calculates district cooling operation for an individual
+of the genetic optimization algorithm and outputs the relevant objective functions.
+"""
+
+import unittest
+import configparser
+import os
+import json
+import tempfile
+import numpy as np
+
+import cea.inputlocator
+import cea.config
+import cea.optimization.preprocessing.preprocessing_main as preprocessing_main
+import cea.optimization.distribution.network_optimization_features as network_optimization_features
+import cea.optimization.slave_data
+import cea.optimization.master.summarize_network as summarize_network
+import cea.optimization.slave.cooling_main as cooling_main
+
+
+class TestDistrictCooling(unittest.TestCase):
+
+    weather_features = None
+    slave_variables = None
+    test_config = None
+    locator = None
+
+    @classmethod
+    def setUpClass(cls, debug=False):
```

Comment:
I don't think the parameter `debug` is required here

Reply:
The debug parameter is not really needed in the function definition, so I removed it there but still left it in the body of the function. I would like to use this parameter to print out certain important intermediary results of the optimisation script in the future.
Would there be a better way to do that?

Reply:
I think having it in the function body is enough for now.